tools/topology/topology2/include/pipelines/cavs/mixout-gain-fir-dai-copier-playback.conf

```diff
@@ -0,0 +1,144 @@
+#
+# BE playback pipeline: mixout-gain-fir-dai-copier.
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.mixout-gain-fir-dai-copier-playback.N.attribute_name"
+#
+# Usage: mixout-gain-fir-dai-copier-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.mixout-gain-fir-dai-copier-playback."N" {
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/gain.conf>
+<include/components/mixout.conf>
+<include/controls/bytes.conf>
+<include/components/eqfir.conf>
+<include/components/pipeline.conf>
+<platform/intel/efx-default.conf>
+
+Class.Pipeline."mixout-gain-fir-dai-copier-playback" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		#
+		# mixout-gain-dai-copier-playback objects instantiated within the same alsaconf node must have
+		# unique pipeline_id attribute
```

Comment:
this doesnt match the attribute name, its instance isnt it?

Reply:
fixed